docs-kits/kits/behaviour-twin-kit/adoption-view/versioning.md

```diff
@@ -0,0 +1,60 @@
+﻿---
+id: versioning
+title: Versioning
+description: Behaviour Twin KIT
+---
+
+<div style={{display:'block'}}>
+  <div style={{display:'inline-block', verticalAlign:'top'}}>
+
+![Behaviour Twin KIT banner](@site/static/img/kit-icons/behaviour-twin-kit-icon-mini.svg)
+
+  </div>
+  <div style={{display:'inline-block', fontSize:17, color:'rgb(255,166,1)', marginLeft:7, verticalAlign:'top', paddingTop:6}}>
+Behaviour Twin KIT
+  </div>
+</div>
+
+## CATENA-X RELEASES
```

Comment:
I would remove this section completely as it is not KIT-specific- Catena-X Release Management is handled in the CX Association - I would broadly defer to the CX Website. 

If you do want to keep it in, please rename "main" into "major".

Reply:
Revised the description and moved it into the *To Be Considered* page.